src/clr-angular/data/datagrid/datagrid-column.ts

```diff
@@ -56,7 +57,7 @@ import { WrappedColumn } from './wrapped-column';
                <ng-container *ngTemplateOutlet="columnTitle"></ng-container>
             </span>
 
-          <clr-dg-column-separator></clr-dg-column-separator>
+          <clr-dg-column-separator *ngIf="!detailService.isOpen"></clr-dg-column-separator>
```

Comment:
Ah this makes it easy. So users won't be able to change the state of a column with detail pane open.

Reply:
+1